packages/customers/src/Nav.tsx

```diff
@@ -27,22 +27,28 @@ const Nav: FC = () => {
   return (
     <>
       <AppNavLink
+        end={!isActive}
         to="customers"
         icon={<Customers />}
         text={t('app.customers')}
       />
       <Collapse in={isActive}>
         <List>
-          <AppNavLink
-            end={true}
-            to="/customers/customer-invoice"
-            text={t('app.customer-invoice')}
-          />
-          <AppNavLink
-            end={true}
-            to="/customers/customer-requisition"
-            text={t('app.customer-requisition')}
-          />
+          {isActive && (
```

Comment:
I'm not sure why these shouldn't be rendered now.. but find to do

Reply:
Ugh I know! I wanted to comment but commenting within the JSX sucks.. it's to avoid a weird animation thing 🙄

Reply:
wondered if it was something like that - just didn't see it when testing. I wasn't as observant as you :)